docker/.env.dist

```diff
@@ -42,3 +42,5 @@ FIREBASE_CLIENT_ID=
 FIREBASE_CLIENT_EMAIL=
 FIREBASE_PRIVATE_KEY=
 FIREBASE_PREFIX=opensalt
+GOOGLE_CAPTCHA_SITE_KEY=
+GOOGLE_CAPTCHA_SECRET_KEY=
```

Comment:
The env variables need to be added to `docker/docker-compose.dev.yml` as well so that they will get passed into the container.